apps/ledger-live-mobile/e2e/page/accounts/accounts.page.ts

```diff
@@ -20,4 +27,9 @@ export default class AccountsPage {
   async addAccount() {
     await tapByElement(this.addAccountButton());
   }
+
+  @Step("Expect accounts number")
```

Comment:
might make sense to log parameter in @step annotation

Reply:
Parameters are automatically logs
<img width="262" alt="Screenshot 2024-12-03 at 08 21 59" src="https://github.com/user-attachments/assets/8de01725-2078-4e6a-b2b6-7b976737b045">